src/CMakeLists.txt

```diff
@@ -41,129 +44,101 @@ else()
-    terminal.h
-    terminal.cc
-    tooltips.h
-    tooltips.cc
-    cmake.h
-    cmake.cc
-    dialogs.cc
-    project.h
     project.cc
+    project.h
     project_build.h
     project_build.cc
-    dispatcher.h
-    dispatcher.cc
-    
+    selectiondialog.cc
+    selectiondialog.h
+    source.cc
+    source.h
+    source_clang.cc
+    source_clang.h
+    terminal.cc
+    terminal.h
+    tooltips.cc
+    tooltips.h
+    window.cc
+    window.h
+
     ../libclangmm/src/CodeCompleteResults.cc
     ../libclangmm/src/CompilationDatabase.cc
     ../libclangmm/src/CompileCommand.cc     
     ../libclangmm/src/CompileCommands.cc    
     ../libclangmm/src/CompletionString.cc   
     ../libclangmm/src/Cursor.cc
+    ../libclangmm/src/Diagnostic.cc
     ../libclangmm/src/Index.cc
     ../libclangmm/src/SourceLocation.cc
     ../libclangmm/src/SourceRange.cc
     ../libclangmm/src/Token.cc
     ../libclangmm/src/Tokens.cc
     ../libclangmm/src/TranslationUnit.cc
-    ../libclangmm/src/Diagnostic.cc
     ../libclangmm/src/Utility.cc
-    
+
     ../tiny-process-library/process.cpp)
 
 if(LIBLLDB_FOUND)
-  list(APPEND source_files debug_clang.h debug_clang.cc)
+  list(APPEND project_files debug_clang.h debug_clang.cc)
 endif()
 
 if(MSYS)
-  list(APPEND source_files dialogs_unix.cc) #dialogs_win.cc does not work any more because of missing SHCreateItemFromParsingName
-  list(APPEND source_files ../tiny-process-library/process_win.cpp)
+  list(APPEND project_files dialogs_unix.cc ../tiny-process-library/process_win.cpp)
 else()
-  list(APPEND source_files dialogs_unix.cc)
-  list(APPEND source_files ../tiny-process-library/process_unix.cpp)
+  list(APPEND project_files dialogs_unix.cc ../tiny-process-library/process_unix.cpp)
 endif()
 
-add_executable(${project_name} ${source_files})
-
-#  add_library(${module} SHARED
-#    api
-#    api_ext)
-
-include_directories(
-  ${Boost_INCLUDE_DIRS}
-#    ${PYTHON_INCLUDE_DIRS}
-  ${GTKMM_INCLUDE_DIRS}
-  ${GTKSVMM_INCLUDE_DIRS}
-  ${LIBCLANG_INCLUDE_DIRS}
-  ${ASPELL_INCLUDE_DIR}
-  ../libclangmm/src
-  ../tiny-process-library
-)
-
-link_directories(
-  ${GTKMM_LIBRARY_DIRS}
-  ${GTKSVMM_LIBRARY_DIRS}
-  ${Boost_LIBRARY_DIRS}
-#    ${PYTHON_INCLUDE_DIRS}
-  ${LIBCLANG_LIBRARY_DIRS}
-)
-
-#  set_target_properties(${module}
-#    PROPERTIES PREFIX ""
-#    LIBRARY_OUTPUT_DIRECTORY "${CMAKE_SOURCE_DIR}/lib/")
-
-#  target_link_libraries(${module} ${PYTHON_LIBRARIES} ${Boost_LIBRARIES})
-#  target_link_libraries(${module} ${Boost_LIBRARIES})
-
-target_link_libraries(${project_name}
-  ${LIBCLANG_LIBRARIES}
-  ${GTKMM_LIBRARIES}
-  ${GTKSVMM_LIBRARIES}
-  ${Boost_LIBRARIES}
-  ${ASPELL_LIBRARIES}
-  ${LIBLLDB_LIBRARIES}
-  #${PYTHON_LIBRARIES}
-)
+include_directories(${global_includes})
+add_executable(${project_name} ${project_files})
+target_link_libraries(${project_name} ${global_libraries})
 
-#  install(TARGETS ${project_name} ${module}
-install(TARGETS ${project_name}
+install(TARGETS ${application_name}
```

Comment:
Make install does not install the juci binary. 

I tried `message("${application_name}")` outputs "" on OS X (only tested here), while `message("${project_name}")` outputs "juci" correctly. I guess you intended to write `install(TARGETS ${project_name}` here or?